app/concepts/todo/create.rb

```diff
@@ -1,50 +1,37 @@
 class Todo::Create < Trailblazer::Operation
+  contract do
+    property :title, validates: {presence: true}
+  end
 
-  # Logic for creating a TODO
-  #     if `todo_list` is specified
-  #         create the `todo`
-  #         associated with `todo_list`
-  #     else
-  #         create the `todo`
-  #         create a `user`
-  #         create a default `todo_list`
-  #         associate `todo_list` with `user`
-  #         associate `todo` with `todo_list`
   def process(params)
-    # make sure title is not blank
-    return invalid! if params[:todo][:title].blank?
+    @model = Todo.new
 
-    # build the todo
-    todo = @model = Todo.new(params.require(:todo).permit(:title, :description))
+    validate(params[:todo], @model) do |f|
```

Comment:
Looking good. This can still be improved a little bit. Try googling for Trailblazer::Operation::Model